lib/build/ProjectBuilder.js

```diff
@@ -388,17 +388,17 @@ class ProjectBuilder {
 		}));
 	}
 
-	async _executeCleanupTasks() {
+	async _executeCleanupTasks(forceCleanup) {
```

Comment:
Hm, here we call it `forceCleanup` and in https://github.com/SAP/ui5-builder/pull/953/commits/2f8181f93d86d7603d853e8f7e625ec58d7d6aa4#diff-58ad88f33b04bd528f373d803ff59e814795eaff5cb0fee7f90cb2732751d294R31 it's `forceTerminate`

Should we just settle with `force`?

```suggestion
	async _executeCleanupTasks(force) {
```